cmake/Config.cmake

```diff
@@ -8,15 +8,15 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     # detect the architecture (note: this test won't work for cross-compilation)
     include(CheckTypeSize)
     check_type_size(void* SIZEOF_VOID_PTR)
-    if("${SIZEOF_VOID_PTR}" STREQUAL "4")
+    if(${SIZEOF_VOID_PTR} STREQUAL "4")
```

Comment:
As far as I know, yes. `STREQUAL` will implicitly interpret both sides as a string and I didn't get any warnings or errors, so I assume it's working.